README.md

```diff
@@ -5,6 +5,10 @@ Oboe is a C++ library which makes it easy to build high-performance audio apps o
 
 [Get started with Oboe here](GettingStarted.md).
 
+# Projects using Oboe
```

Comment:
Add a line requesting developers to contact us if they have an Oboe app in the Play Store.

Reply:
Done